src/main/scala/com.snowplowanalytics/sqs2kinesis/config/CliConfig.scala

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2020-2021 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+
+package com.snowplowanalytics.sqs2kinesis.config
+
+import cats.syntax.either._
+import cats.syntax.show._
+import io.circe.config.parser
+import com.monovore.decline.{Command, Opts}
+import com.typesafe.config.{Config, ConfigFactory}
+
+import com.snowplowanalytics.sqs2kinesis.generated.BuildInfo
+
+import java.nio.file.Path
+import scala.io.Source
+
+case class CliConfig(app: Sqs2KinesisConfig, raw: Config)
+
+object CliConfig {
+
+  val configOpt: Opts[Option[Path]] = Opts.option[Path]("config", "Path to configuration hocon file").orNone
```

Comment:
Shouldn't the path be required ?

Reply:
There are occasions when it is not easy to pass a config file to this app; for example if it is launched on a container engine without a mounted volume.  In those situations the config parameters would be passed with system parameters, instead of a config file.  See [the example in the readme](https://github.com/snowplow-incubator/sqs2kinesis/blob/feature/max-batch-size/README.md#quickstart)